components/calendar/src/astronomy.rs

```diff
@@ -979,6 +979,8 @@ impl Astronomical {
     }
 
     #[allow(dead_code, clippy::unwrap_used, clippy::eq_op)]
+    // Time between sunset and moonset on date at location. Returns None if there is no sunset or moonset on said date.
```

Comment:
nit: comments go above the annotations. change in this file & in islamic.rs

(that keeps the code together with other code)

Reply:
Done in both files, thanks